Sources/SwiftDocC/Utility/FoundationExtensions/String+Capitalization.swift

```diff
@@ -12,37 +12,24 @@ import Foundation
 
 extension String {
     
+    /// Precomputes the CharacterSet to use in capitalizeFirstWord().
```

Comment:
nit: this is private API so doc comments don't get forwarded through. Use a regular comment here.